### PR TITLE
e2e: Killed gossip e2e tests.

### PR DESCRIPTION
**Pull request by bwplotka**
:fire: :fire: 

Signed-off-by: Bartek Plotka <bwplotka@gmail.com>

### CODE REVIEW

**Comment by FUSAKLA** [comment]
Just out of curiosity what is the timeline to drop the gossip?

File: test/e2e/spinup_test.go [hunk 1]
```diff
@@ -100,28 +88,19 @@ func scraper(i int, config string, gossip bool) (cmdScheduleFunc, string) {
...
-			args = append(args, []string{
-				"--cluster.advertise-address", sidecarCluster(i),
-				"--cluster.gossip-interval", "200ms",
-				"--cluster.pushpull-interval", "200ms",
-			}...)
-			args = append(args, clusterPeerFlags...)
-		} else {
-			args = append(args, "--cluster.disable")
-		}
+		args = append(args, "--cluster.disable")
```
Comment (ivan-valkov) [inline test/e2e/spinup_test.go]: we don't need this and test fail because we have repeating `--cluster.disable`

Comment (bwplotka) [reply test/e2e/spinup_test.go]: and that what actually happen in CI..

**Comment by bwplotka** [comment]
@FUSAKLA 
> Just out of curiosity what is the timeline to drop the gossip?

Not sure yet, I know people still use it. I think we will do next release as the last one with gossip.

**Pull request merged**